docs/dictionary/message/newWidget.lcdoc

```diff
@@ -0,0 +1,31 @@
+Name: newWidget
+
+Type: message
+
+Syntax: newWidget
+
+Summary: Sent to a new <widget> right after it's created.
+
+Associations: widget
+
+Introduced: 8.0
+
+OS: mac,windows,linux,ios,android
+
+Platforms: desktop,server,web,mobile
+
+Example:
+on newWidget -- set tooltips of new widgets
+	set the tooltip of the target to the kind of the target
+end newButton
+
+Description:
+Handle the <newWidget> <message> if you want to create additional <object|objects>, switch <tool|tools>, or do other tasks when a new <widget> is created.
+
+Normally, the <newWidget> <message> is handled at a higher level of the <message path>, since the newly created <widget> does not yet have a <script>.
+
+The actual creation is not triggered by the <newWidget> <message>, so <trap|trapping> the <message> and not allowing it to <pass> does not prevent the <widget> from being created.
```

Comment:
It could perhaps be clearer to say that the `newWidget` is sent *after* the widget is created.